components/Carousel/Carousel.js

```diff
@@ -1,5 +1,42 @@
 class Carousel {
+  constructor(element) {
+    this.element = element;
 
+    // Puts Carousel Items into an array
+    this.items = element.querySelectorAll(".Carousel__item");
+    this.items = Array.from(this.items);
+    console.log(this.items);
```

Comment:
I'd suggest to remove console.log on completed projects. It is okay to have them while working on the code, but once you finish it, it's better to remove them.